screenfetch-dev

```diff
@@ -1847,7 +1847,7 @@ detectmem () {
 		#done
 		#usedmem=$((usedmem / 1024))
 		#totalmem=$((totalmem / 1024))
-		mem=$(free -b | awk -F ':' 'NR==2{print $2}' | awk '{print $1"-"$6}')
+		mem=$(free -b | awk -F ':|：' 'NR==2{print $2}' | awk '{print $1"-"$6}')
```

Comment:
This change could ensure output is desired under zh_CN and possibly zh_TW, zh_HK, zh_SG, etc. But what about other locales? Output for `free -b` might vary much for other locales and apparently screenfetch cannot ensure it can parse those outputs in a few lines' workaround.
Instead of trying to parse locale-specific outputs, I would rather suggest forcing `free -b` to output in en_US locale and just parse that output. e.g.
```
mem=$(LANG=en_US free -b | awk -F ':' 'NR==2{print $2}' | awk '{print $1"-"$6}')
```